### DISCUSSION TITLE
[Feature Request] Add more keys (WASD)

### DISCUSSION THREAD

**Discussion by Smurr** [cafali/SnapKey#20] [New Features & Ideas]
Instead of just A or D to be changed, i would like to use W and S too without having to change A and D. Maybe a possibility to add keys on the config file like keyW=x. I really like the app. I was using AHK before but this looks really good and clean.

**Comment by minteeaa** [comment]
definitely possible, but i wonder if the idea of extending Snap Tap to (theoretically) all keyboard keys is going a bit out of bounds with the original feature's idea?

**Reply by Smurr** [reply]
Only movement keys would be great, WASD.

**Reply by Agitatio** [reply]
Wooting already added that to their software like a week ago, for all of their keyboards and with support for multiple key pairs. There's also Free Snap Tap that has multiple pairs functionality and works in more games.

**Comment by cafali** [comment]
> Instead of just A or D to be changed, i would like to use W and S too without having to change A and D. Maybe a possibility to add keys on the config file like keyW=x. I really like the app. I was using AHK before but this looks really good and clean.

All four movement keys (W, A, S, D) are now supported by default in the latest release of SnapKey (1.1.8) and can be rebound using the configuration file.


SnapKey 1.1.8 - 03/08/2024

SnapKey:
- Multi-Key Support: SnapKey now supports simultaneous use of up to 4 keys. 
- The keys are divided into two independent groups:
  - Group 1: Key1 and Key2
  - Group 2: Key3 and Key4

- Independent Key Handling: Keys from **Group 1** (Key1 & Key2) operate independently of keys from **Group 2** (Key3 & Key4). 

Pressing a key from Group 1 will not automatically cancel a key from Group 2.

Default Key Configuration (A/D & W/S):
- Key1=65
- Key2=68
- Key3=83
- Key4=87

Others:
- Updated config.cfg
- Updated README.pdf
- Updated backup.snapkey
